modules/search_api_pantheon_examples/search_api_pantheon_examples.module

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Module file for search_api_pantheon_examples.
+ */
+
+/**
+ * Implements hook_search_api_solr_config_files_alter().
+ */
+function search_api_pantheon_examples_search_api_solr_config_files_alter(array &$files, string $lucene_match_version, string $server_id = '') {
+  // Append at the end of the file.
+  $files['solrconfig_extra.xml'] .= "<writeLockTimeout>500</writeLockTimeout>";
```

Comment:
Let's show parsing of the schema.xml with a simple XML object and changing some value in the schema.xml

Reply:
Done!

Reply:
Great work!